### DISCUSSION TITLE
Casting feature!!!

### DISCUSSION THREAD

**Discussion by hattbsdk** [pear-devs/pear-desktop#2359] [Ideas]
Can we please have the ability to cast the music, like the PWA and online YTMusic have? It would really make a big difference as I would want to cast it to my speakers with the fantastic interface and features. 

Sorry I'm very new here and to GitHub so if this feature already exists, please let me know!

**Comment by ArjixWasTaken** [comment]
https://github.com/th-ch/youtube-music/issues/113
